### PR TITLE
Deimplement token generation and Identity services

### PR DESCRIPTION
**Pull request by MahdiyarGHD**
### Checklist:

- [ ] It refers to an [Issue](https://github.com/EasyMicroservices/FileManager/issues).
- [ ] It fixes a bug.
- [ ] It's a new feature (non-breaking change which adds functionality)
- [ ] It's a breaking (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] CHANGELOG is updated.
- [ ] Related language label is selected.

### CODE REVIEW

File: src/CSharp/EasyMicroservices.AuthenticationsMicroservice.Services/JWTManager.cs [hunk 1]
```diff
@@ -1,89 +1,89 @@
...
-using EasyMicroservices.ServiceContracts;
-using Microsoft.Extensions.Configuration;
-using Microsoft.IdentityModel.Tokens;
-using System;
-using System.IdentityModel.Tokens.Jwt;
-using System.Linq;
-using System.Security.Claims;
-using System.Text;
-using System.Threading.Tasks;
+﻿//using EasyMicroservices.AuthenticationsMicroservice.Contracts.Common;
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.AuthenticationsMicroservice.Services/JWTManager.cs]: Please remove unused codes

File: src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs [hunk 2]
```diff
@@ -15,104 +15,116 @@ namespace EasyMicroservices.AuthenticationsMicroservice.WebApi.Controllers
...
-        }
-
-        [HttpGet]
-        [Authorize]
-        public string Test()
-        {
-            return "test";
-        }
+        //[HttpPost]
+        //public async Task<MessageContract<bool>> VerifyUserName(VerifyEmailAddressContract request)
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs]: Please remove unused codes

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by Hamed-Bagherpour — APPROVED** [review]

**Review by Elyas-Amini — APPROVED** [review]

**Review by mahdidelzendeh72 — APPROVED** [review]
thanks

File: src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs [hunk 3]
```diff
@@ -24,13 +24,13 @@ public UsersController(IUnitOfWork unitOfWork/*, IJWTManager jwtManager*/) : bas
         }
 
         [HttpPost]
-        public async Task<MessageContract> UserHasExistsByUsername(UserHasExistsByUsernameRequestContract request)
+        public async Task<MessageContract<UserContract>> UserHasExistsByUsername(UserHasExistsByUsernameRequestContract request)
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs]: It's not better to change method name too "GetUserByUserName" ?

File: src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs [hunk 4]
```diff
@@ -24,13 +24,13 @@ public UsersController(IUnitOfWork unitOfWork/*, IJWTManager jwtManager*/) : bas
...
         {
             return await _unitOfWork.GetContractLogic<UserEntity, AddUserRequestContract, UserContract, UserContract, long>().GetBy(x => x.UserName == request.Username);
         }
                
         [HttpPost]
-        public async Task<MessageContract> VerifyUserIdentity(UserSummaryContract request)
+        public async Task<MessageContract<UserContract>> VerifyUserIdentity(UserSummaryContract request)
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs:30]: Why we should return UserContract in veryfiy?

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

File: src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs (hunk 3 above)
Comment (MahdiyarGHD) [reply src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs]: Sure

**Review by AmirmohammadNemati — APPROVED** [review]

**Review by Ali-YousefiTelori — APPROVED** [review]

**Review by mahdidelzendeh72 — COMMENTED** [review]
thanks

**Pull request merged**